js/samples/menu-example/rag/README.md

```diff
@@ -0,0 +1,41 @@
+# Evaluating menuQA flow
+
+## Build it
+
+```
+pnpm build
+```
+
+or if you need to, build everything:
+
+```
+cd ../../../; pnpm build; pnpm pack:all; cd -
```

Comment:
is there a better way we can say "go to root" instead of '../../../ ?

Reply:
Not sure, but we actually use it in a bunch of samples e.g. byo-evaluator, cat-eval, evaluator-gut-check so atleast it's consistent 🤷